docs/spark/tutorials/get-started.md

```diff
@@ -39,39 +39,37 @@ Install [Java 8.1](https://www.oracle.com/technetwork/java/javase/downloads/jdk8
 
 Select the appropriate version for your operating system. For example, select **jdk-8u201-windows-x64.exe** for a Windows x64 machine. Then, use the command `java` to verify the installation.
 
-![Java Download](https://dotnet.microsoft.com/static/images/java-jdk-downloads-windows.png?v=6BbJHoNyDO-PyYVciImr5wzh2AW_YHNcyb3p093AwPA)
+![Java Download](https://dotnet.microsoft.com/static/images/java-jdk-downloads-windows.png)
```

Comment:
Is it okay to have images from links like this ? Or they should be ported into this repo to a media directory ?